new-log-viewer/src/components/StatusBar/index.tsx

```diff
@@ -37,8 +38,12 @@ const StatusBar = () => {
             >
                 Status message
             </Typography>
+            <LogLevelSelect/>
             <Button
                 size={"sm"}
+                sx={{
+                    borderRadius: 0,
```

Comment:
Might be good to create a ".borderless" CSS class and set the `className` of this Button to be `borderless`. What do you think?